src/main/java/org/basseur/adventofcode/advent2018/days/day06/Day06.java

```diff
@@ -0,0 +1,199 @@
+package org.basseur.adventofcode.advent2018.days.day06;
+
+import org.basseur.adventofcode.advent2018.ProblemStatusEnum;
+import org.basseur.adventofcode.advent2018.days.Days;
+import org.basseur.adventofcode.advent2018.utils.FileReaders;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Implementation for <i>Day 6: Chronal Coordinates</i>.
+ *
+ * @author Jan Philipp G&ouml;lz
+ */
+@Component
+public class Day06 implements Days {
+
+    /** The location of the puzzle input file */
+    private static final String FILE_LOCATION = "src/main/java/org/basseur/adventofcode/advent2018/days/day06/Input.txt";
+    /** The puzzle status {@code HashMap} */
+    private final HashMap<String, ProblemStatusEnum> problemStatus;
+    /** The list of coordinate Strings */
```

Comment:
Only a general comment, as I know some co workers might also have commented on this (I'm myself not a fan on being picky about this things in pull requests though): The name of your attribute says already what the comment says. So in a way, this is redundant information. 
Or: dos it get described in a JavaDoc afterwards if you comment it like this?

Reply:
Yes, this is only for the JavaDocs. I don't know how useful JavaDocs really are in this project (probably not at all).